code/modules/admin/admin.dm

```diff
@@ -91,12 +91,14 @@ GLOBAL_VAR_INIT(nologevent, 0)
 		else
 			body += "\[[M.client.holder ? M.client.holder.rank : "Player"]\] "
 		body += "\[<A href='?_src_=holder;getplaytimewindow=[M.UID()]'>" + M.client.get_exp_type(EXP_TYPE_CREW) + " as [EXP_TYPE_CREW]</a>\]"
+		body += "<br>BYOND account registration date: [M.client.byondacc_date || "ERROR"]<br>"
```

Comment:
Putting the raw date in here isn't super helpful.
Instead, show how many days old their byond account is.
E.g: BYOND account age: 0 days
BYOND account age: 394 days
etc.
Perhaps hide this info if their account is over ... 365d old.
And bold it if their account is <7d old.

Reply:
Ill add some logic to work out how many days old, as well as bold it if it is under 7